fabric-mining-levels-v0/src/main/java/net/fabricmc/fabric/mixin/tools/MixinItemStack.java

```diff
@@ -48,8 +48,12 @@ public void getBlockBreakingSpeed(BlockState state, CallbackInfoReturnable<Float
 			TriState triState = ToolManager.handleIsEffectiveOn((ItemStack) (Object) this, state);
 			if (triState != TriState.DEFAULT) {
 				float miningSpeed;
-				if (this.getItem() instanceof DynamicTool) miningSpeed = ((DynamicTool) this.getItem()).getMiningSpeed((ItemStack)(Object) this);
-				else miningSpeed = ((MiningToolItemAccessor) this.getItem()).getMiningSpeed();
+				if (this.getItem() instanceof DynamicTool) {
+					miningSpeed = ((DynamicTool) this.getItem()).getMiningSpeed((ItemStack)(Object) this);
+				}
+				else {
```

Comment:
Ouch! My back!

Did you intend to use a fractured-else here?

Reply:
> Fractured-else?
It's my name for when the style changes mid-else. I see it a lot at work. Some people actually prefer it, believe it or not.